### PR TITLE
update the dial interface to cope with new pull additions

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,12 +94,7 @@ This method dials a transport to the Peer listening on `multiaddr`.
 
 `callback` should follow the `function (err, conn)` signature.
 
-`conn` is the same `conn` that gets returned by call, which should follow [`interface-connection`](https://github.com/diasdavid/interface-connection). This `conn` object can emit 3 extra events:
```
Comment (dignifiedquire) [inline README.md]: The callback now has the signature of only `function (err)` not with the `conn` inside

Comment (diasdavid) [inline README.md]: We don't want to leave the conn inside too?

Comment (dignifiedquire) [inline README.md]: It's not at this point at least

Comment (diasdavid) [inline README.md]: I would like to keep it there too, it is convenient when used with waterfall flows

File: package.json [hunk 2]
```diff
@@ -15,9 +30,12 @@
     "url": "https://github.com/diasdavid/interface-transport/issues"
   },
   "homepage": "https://github.com/diasdavid/interface-transport",
-  "devDependencies": {},
+  "devDependencies": {
+    "aegir": "^7.0.1"
```
Comment (diasdavid) [inline package.json]: Is aegir 7 working?

**Comment by diasdavid** [comment]
I see you added tests 👍 now it needs a rebase :)

File: package.json (hunk 2 above)
Comment (dignifiedquire) [inline package.json]: sometimes :D

**Pull request merged**